### PR TITLE
Bigger upload size

### PR DESCRIPTION
**Pull request by bastbijl**
Use stream parser in sdk for importing gzipped write operations

### CODE REVIEW

File: src/main/java/com/weaverplatform/postgresql/util/File.java [hunk 1]
```diff
@@ -27,8 +27,8 @@
   public static String LOCATION;
   
   static {
-    MAX_FILE_SIZE           = Long.parseLong(Props.get("file.max_file_size"));    // the maximum size allowed for uploaded files
-    MAX_REQUEST_SIZE        = Long.parseLong(Props.get("file.max_request_size")); // the maximum size allowed for multipart/form-data requests
+    MAX_FILE_SIZE           = Long.parseLong(Props.get("MAX_FILE_SIZE","file.max_file_size"));    // the maximum size allowed for uploaded files
+    MAX_REQUEST_SIZE        = Long.parseLong(Props.get("MAX_REQUEST_SIZE", "file.max_request_size")); // the maximum size allowed for multipart/form-data requests
```
Comment (gvdent) [inline src/main/java/com/weaverplatform/postgresql/util/File.java]: This is new... does it allow the size's to be defined through environment variables?

File: src/main/java/com/weaverplatform/postgresql/controllers/NodeController.java [hunk 2]
```diff
@@ -59,24 +61,21 @@
    * 6min for this zip to execute on a 2016 Macbook Pro.
    */
   public static Route uploadZip = (Request req, Response res) -> {
+    final int CHUNK_SIZE = 100000;
```
Comment (gvdent) [inline src/main/java/com/weaverplatform/postgresql/controllers/NodeController.java]: Could this be made configurable please? I can imagine it requiring some tweaking depending on the system it's running on, that should not require a rebuild.

File: src/main/java/com/weaverplatform/postgresql/controllers/NodeController.java [hunk 3]
```diff
@@ -1,29 +1,31 @@
...
+import java.io.InputStream;
+import java.util.List;
+import java.util.zip.GZIPInputStream;
+
 /**
  * @author Mohamad Alamili
  */
 public class NodeController {
 
+  static Logger logger = LoggerFactory.getLogger(Application.class);
```
Comment (gvdent) [inline src/main/java/com/weaverplatform/postgresql/controllers/NodeController.java]: This should be `getLogger(NodeController.class)`, that will also get rid of the import Application.

File: src/main/java/com/weaverplatform/postgresql/controllers/NodeController.java [hunk 4]
```diff
@@ -1,29 +1,31 @@
 package com.weaverplatform.postgresql.controllers;
 
-import java.io.BufferedWriter;
-import java.io.FileWriter;
-import java.io.InputStream;
-import java.util.List;
-import java.util.zip.GZIPInputStream;
-
 import com.google.gson.Gson;
+import com.weaverplatform.postgresql.Application;
```
Comment (gvdent) [inline src/main/java/com/weaverplatform/postgresql/controllers/NodeController.java]: Que? This should never be needed anywhere.

File: src/main/java/com/weaverplatform/postgresql/util/File.java (hunk 1 above)
Comment (bastbijl) [reply src/main/java/com/weaverplatform/postgresql/util/File.java]: yep, agree?

Comment (bastbijl) [reply src/main/java/com/weaverplatform/postgresql/util/File.java]: is the docker name sysunite/weaver-database-postgresql:0.0.29-SNAPSHOT-bigger-upload-size ok?

Comment (gvdent) [reply src/main/java/com/weaverplatform/postgresql/util/File.java]: perfect :)

**Comment by gvdent** [comment]
Approved pending succesfull weaver-sdk-js integration test

**Pull request merged**